packages/react/src/components/button.tsx

```
@@ -91,247 +75,119 @@
+            case "uploading": {
+              if (uploadProgress === 100) return <Spinner />;
+              return (
+                <span className="z-50">
+                  <span className="block group-hover:hidden">
+                    {uploadProgress}%
+                  </span>
+                  <Cancel cn={cn} className="hidden size-4 group-hover:block" />
+                </span>
+              );
+            }
+            case "disabled":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)